saad/sprint1/infra/lambda/.~c9_invoke_g7YmyU.py

```diff
@@ -0,0 +1,31 @@
+import urllib3
+from time import time
+import web_constants
+
+def lambda_handler(event,context):
+    message=read_url_FromFile('website_list.txt')
+    return message
+
```

Comment:
Consider explaining input and output arguments through comments for this function

Reply:
By the way I don't know why this file was created It should not be here I will remove it. The only file containing the lambda handler is web_metrics.py